web_src/js/features/repo-legacy.js

```diff
@@ -559,7 +560,7 @@ function initRepoIssueCommentEdit() {
 
   // Quote reply
   $(document).on('click', '.quote-reply', function (event) {
-    $(this).closest('.dropdown').find('.menu').toggle('visible');
+    $(this).closest('.dropdown').find('.menu').toggle('visible'); // eslint-disable-line
```

Comment:
Why not `toggleElem`?

Reply:
This `toggle('visible');`  is not a general `toggle` function.

I am not sure what it is for, or whether it's Fomantic-UI related. So I chose to keep it as it was. Leave to next round.

Reply:
I think I know what the `toggle('visible')` means.

jQuery has the `toggle(easing)` animation support. Usually it's used like this: `$e.toggle('fast')` or `$e.toggle('slow')`. However, if the toggle function gets an invalid string `$e.toggle('anything-wrong')`, it still uses a default easing animation. IMO the `visible` is a misused name.

So, I think it safe to replace the `toggle(easing)`  by our `toggleElem`, because that animation isn't useful on the review page, instead, the animation makes users feel laggy.

I will refactor the `toggle('visible')` in next PR, to make it easier to be tested.